runtime/compilers/rillv1/parse_rillyaml.go

```diff
@@ -60,6 +63,9 @@ func (p *Parser) parseRillYAML(ctx context.Context, path string) error {
 	}
 
 	for i, c := range tmp.Connectors {
+		if _reservedConnectorNames[c.Name] {
+			return fmt.Errorf("%s are reserved connector names", maps.Keys(_reservedConnectorNames))
```

Comment:
I thought it is easier for user if we return all the reserved names in one go.

Reply:
I think the error message should also specifically identify the problem. Like if you call a table "from" in SQL, it will say something like "unexpected keyword 'from'", not "the reserved keywords are [.......]".